duckduckgo/iOS
# Better refelect unprotected status

Task/Issue URL: https://app.asana.com/0/0/1200517965702748/f
Tech Design URL:
CC: @brindy 

**Description**:
This PR is a follow up to handling temporarily unprotected sites. This PR shows the site as unprotected when viewing the privacy dashboard in addition to the previous "trackers detected" fix.

**Steps to test this PR**:
1. Visit https://spiegel.de
1. Open privacy dashboard
2. Site should appear as unprotected
3. Toggling protections will immediately disable them



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
